app/views/hyku_addons/account_settings/index.html.erb

```diff
@@ -19,36 +19,34 @@
 
     <% ['contact_email', 'gtm_id', 'demo_gtm_id', 'live', 'institutional_relationship', 'creator_fields', 'contributor_fields'].each do |fname| %>
```

Comment:
We could extract this into a rails helper that accepts a block. That would improve readability which is extremely important on the views IMO.

Something like:
```

def for_available_setting_field
  Account.contact_settings_fields.each do |field|
    capture(yield(field))
  end
end
```

And then use it as:

```
<% for_available_setting_field do |field | %>
....

```

Reply:
I agree but I'm not sure it's something I want to spend time on now